File: week08/sample19-properties/java/Account.java
public class Account {
    private static int numberOfAccounts = 0;
    public static int getNumberOfAccounts() { return numberOfAccounts; }
    private final long created;
    public Account() {
        numberOfAccounts++; // <=> Account.numberOfAccounts++;
        created = System.currentTimeMillis(); // <=> this.created = System.currentTimeMillis();
    }
    public long getCreated() { return created; }
}

File: week08/sample19-properties/Account.kt
class Account {
    private val created: Long
        get() = field // <=> get() { return field }

    init { // Initializer block: is called after the primary constructor
        Account.numberOfAccounts++ // <=> Account.numberOfAccounts++
        created = System.currentTimeMillis()
    }

    companion object {
        var numberOfAccounts = 0
            private set // excludes the setter method
    }
}